app/services/platforms/youtube.ts

```diff
@@ -57,6 +57,13 @@ export class YoutubeService extends Service implements IPlatformService {
     });
   }
 
+  fetchDescriptionString(): Promise<string> {
```

Comment:
I would call this `fetchDescription`.  The nice thing about typescript is you already know it returns a string.

Reply:
Good Point